### DISCUSSION TITLE
Is dZ3 enabled by default?

### DISCUSSION THREAD

**Discussion by heroinedd** [Z3Prover/z3#7360] [Q&A]
Hi,

I read from dZ3's paper that it is integrated with Z3. However, I wonder if it is used as the regex solver by default. Or do I have to specify some command line options to use it, like appointing `smt.string_solver=z3str3` if I want to use z3str3 as the string solver?

Thanks!

**Comment by NikolajBjorner** [comment]
z3str3 doesn't use dZ3. It uses automata.

**Reply by heroinedd** [reply]
So is there any string solver of z3 uses dZ3?
